RxCocoa/Foundation/URLSession+Rx.swift

```diff
@@ -228,3 +228,13 @@ extension Reactive where Base: URLSession {
     }
 }
 
+extension Reactive where Base == URLSession {
+/// Log URL requests to standard output in curl format.
+   public static var shouldLogRequest: (URLRequest) -> Bool = { _ in
+	   #if DEBUG
+		   return true
+	   #else
+		   return false
+	   #endif
+   }
+}
```

Comment:
Indentation seems wrong here
```suggestion
    /// A closure to determine if cURL-formatted request information shroud be loged
    /// for a specific `URLRequest`
    public static var shouldLogRequest: (URLRequest) -> Bool = { _ in
        #if DEBUG
            return true
        #else
            return false
        #endif
    }
}
```